### PR TITLE
ci(runner): Check Invalidation Complete before Triggering Site Workflow

### LINKED ISSUE #14: Check Status of Cloudfront Invalidation Before Triggering Workflow
### Problem

Before triggering the open-sdg-site-starter workflow, check the status of the Cloudfront Invalidation.

Resources: https://github.com/swoodford/aws

### PR DESCRIPTION
**Pull request by tmargono**
### What does this MR do?

- Check Invalidation is Complete before Triggering Site Workflow
- Make necessary changes to prod pipeline to make it identical to dev pipeline

### Background info

Script used is a modified, simplified version of https://github.com/swoodford/aws/blob/master/cloudfront-invalidation-status.sh.

IAM policy for user was modified to include the permissions cloudfront:ListInvalidations , cloudfront:GetInvalidation necessary for the script.

### How can this be tested (manually and/or automated test)?

Tested successfully. https://github.com/CityOfLosAngeles/open-sdg-data-starter/actions/runs/677188608

### Which issue(s) is/are related to this PR?

This PR is/are related to issue(s) issue #14

close #14

### CODE REVIEW

File: scripts/check_invalidations.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	if [ -z "$invalidations" ]; then
+		echo No CloudFront Invalidations In Progress.
+		HorizontalRule
+		return 1
+	else
+		# Loop through all invalidations
+		while IFS= read -r invalidationid
+		do
+			echo "Waiting for invalidation $invalidationid to complete..."
+			# Poll every 20 seconds, Wait until invalidation has completed
```
Comment (jaylenw) [inline scripts/check_invalidations.sh]: Hey @tmargono I am not clear where the 20 seconds is coming from. May you elaborate on this and possible explain in the comment on this line as well? Thanks.

**Review by jaylenw — COMMENTED** [review]
Thanks for cleaning up the `yml` files. The addition of the script to verify if the cloud distribution has been completed before triggering the workflow in the site starter repository is great.

File: scripts/check_invalidations.sh (hunk 1 above)
Comment (tmargono) [reply scripts/check_invalidations.sh]: The 20s poll is built into the AWS command itself, we do not need to implement it. I have updated the comment with more info, see https://docs.aws.amazon.com/cli/latest/reference/cloudfront/wait/invalidation-completed.html.

Comment (jaylenw) [reply scripts/check_invalidations.sh]: Ok cool!

**Review by jaylenw — APPROVED** [review]

**Pull request merged**